src/Orleans.Core/Core/ClientBuilder.cs

```diff
@@ -41,10 +60,24 @@ public IClusterClient Build()
         }
 
         /// <inheritdoc />
-        public IClientBuilder ConfigureServices(Action<IServiceCollection> configureServices)
+        public IClientBuilder ConfigureHostConfiguration(Action<IConfigurationBuilder> configureDelegate)
```

Comment:
Why change this name?  ConfigureHostConfiguration seems to indicate that this is for a host, which is not the case, correct?  Can't multiple clients can be configured on the same host?  Isn't this only for a specific client?

Reply:
It's just bad diffing, I'm not *changing* the name.
re: ~"multiple clients configured on the same host", there's not really any notion of a host here

This method configures `IConfiguration` for use in other builder calls such as `ConfigureServices` and `ConfigureAppConfiguration`. Maybe the method should be renamed `ConfigureBuilderConfiguration`?